app/src/app/store/mapStore.ts

```diff
@@ -9,8 +9,13 @@ import {
 } from "../api/apiHandlers";
 import maplibregl from "maplibre-gl";
 import type { MutableRefObject } from "react";
-import { addBlockLayers } from "../constants/layers";
+import {
+  addBlockLayers,
```

Comment:
I added some notes but I don't think we should be augmenting the map from the store- we can `subscribe` to changes and update that way but shouldn't call non-store functions inside

Reply:
Makes total sense and the subscribe approach is much nicer